src/actions/editUserActions.js

```diff
@@ -0,0 +1,171 @@
+let { writeFile, readFile } = require("../helpers/helpers")
+
+//metoda folosita in reqValidData pentru a verifica daca mailul deja exista
+let checkMail = (newMail, allMails) => {
+    for (let i = 0; i < allMails.length; i++) {
+        if (newMail === allMails[i].email) return 1;
+    }
+    return 0;
+};
+
+//metoda verifica daca elementele introduse de user sunt corecte si corespund campului corespunzator
+let reqValidData = (str, putMethod,users) => {
```

Comment:
These does something very similar with **dataValid** function from **createUserActions.js**. Please refactor before merge or create a new task in trello for this refactor

Reply:
Nu putem folosi functia dataValid din createUserActions.js pentru ca noi am implementat promise-uri si astfel apar discrepante in cod